### ISSUE TITLE
Two issues in Transformer Experiments

### ISSUE THREAD

**Issue by HMJiangGatech** [tbachlechner/ReZero-examples#4]
1. In the forward function of ReZeroEncoderLayer
src2 = self.linear2(self.dropout(self.activation(self.linear1(src))))
should be 
src2 = self.linear2(self.dropout(self.activation(self.linear1(**src2**))))

2. The learning rate of training with post/pre-lm and rezero are different:
Post/Pre-lm: lr = 0.1
Rezero: lr = 0.4

By fixing the above two issues, pre-lm + lr=0.4 is faster than rezero on wikitext-2 data:
`| end of epoch   3 | time: 34.79s | valid loss  5.11 | valid ppl   165.21`

**Comment by tbachlechner** [comment]
Thank you for carefully checking the code and for pointing out this bug. I updated the notebook and instead of demonstrating ReZero at a 6 layer network I now use a deeper 12 layer network.
